docs/assets/js/main.js

```diff
@@ -46,6 +46,16 @@ function isInRange(num, min, max) {
 }
 
 function navigateBack() {
+    const currentHost = window.location.host;
+    const referrer = document.referrer;
+
+    // Check if the referrer is the same as the current host
```

Comment:
I would prefer a larger comment at the top that explains why the code is the way it is, not what it is. Why do check that referrer is the same as current host, and why does that result in us using browser history?